versioning_base.py

```diff
@@ -1092,8 +1121,80 @@ def add_branch( pg_conn_info, schema, branch, commit_msg,
     pcur.commit()
     pcur.close()
 
+def diff_rev_view_str(pg_conn_info, schema, table, branch, rev_begin, rev_end):
+    """DIFFerence_REVision_VIEW_STRing
+    Create the SQL view string of the specified revision difference (comparison).
+    """
+    pcur = Db(psycopg2.connect(pg_conn_info))
+
```

Comment:
Adding:
```
rev_begin_, rev_end_ = str(rev_begin), str(rev_end)
```
would IMO increase readability of the following code in this function